eis_toolkit/checks/pixel_alignment.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+import rasterio
+
+from eis_toolkit.checks.cell_size import check_matching_cell_size
+
+
+def check_matching_pixel_alignment(
+    rasters: List[rasterio.io.DatasetReader],
+) -> bool:
+    """Check if every raster in a list has matching cell size and matching pixel alignment.
+
+    Args:
+        rasters: List of rasters to check.
+
+    Returns:
+        bool: True if cell size and pixel alignment matches, false if not.
```

Comment:
You could have all the boolean values with capital letter, as was in other part of your comments.